cupy/core/_reduction.pyx

```diff
@@ -506,7 +506,8 @@ cdef class _SimpleReductionKernel(_AbstractReductionKernel):
 
     cdef:
         readonly _kernel._Ops _ops
-        readonly _preamble
+        readonly str preamble
+        readonly str _preamble
```

Comment:
@emcastillo Let me try merging these two to `preamble`. The underscored version was used in fusion, but in ReductionKernels the one without the underscore is used.

Reply:
no prob! thanks

Reply:
Done in 514042f.